blocks/career-carousel/career-carousel.js

```diff
@@ -0,0 +1,113 @@
+import { fetchIndex } from '../../scripts/scripts.js';
+
+function filterIncompleteEntries(json) {
+  return json.data.filter((e) => e.image !== '' && e['career-quote'] !== '0' && e['career-jobtitle'] !== '0');
+}
+
+function scrollToCard(idx, card, precedingCard, slides, span) {
+  const rect = card.getBoundingClientRect();
+
+  // set the style on the active button
+  const buttons = document.querySelectorAll('.career-slides-nav span');
+  buttons.forEach((b) => b.classList.remove('active-nav'));
+  span.classList.add('active-nav');
+
+  // Compute the gap to add to the location
+  let gap = 0;
+  if (precedingCard) {
+    const prevRect = precedingCard.getBoundingClientRect();
+    gap = rect.x - (prevRect.x + prevRect.width);
+  }
+
+  slides.scrollTo(idx * (rect.width + gap), slides.scrollHeight);
+}
+
+export default async function decorate(block) {
+  const json = await fetchIndex('query-index', 'career-testimonials');
+  const data = filterIncompleteEntries(json);
+
+  const careerSlider = document.createElement('div');
+  careerSlider.classList.add('career-slider');
+
+  const careerSlides = document.createElement('div');
+  careerSlides.classList.add('career-slides');
+
+  const slideDivs = [];
+  for (let i = 0; i < data.length; i += 1) {
+    const div = document.createElement('div');
+    div.classList.add('career-card');
+    const a = document.createElement('a');
+    a.href = data[i].path;
+    const fig = document.createElement('figure');
+    const img = document.createElement('img');
```

Comment:
Should we use https://github.com/hlxsites/sunstar/blob/a634b9e3cf1d6f052f212ecfdae2e9eada2a2b91/scripts/lib-franklin.js#L497-L537 for `picture`  tags?

Reply:
Thanks for the suggestion. I have made this change now.